pyproject.toml

```
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "epcsunspecdemo"
+version = "0.1.0.post46.dev0+2a57176"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+
+[tool.poetry-dynamic-versioning]
+enable = true
+
+
+[tool.poetry.dependencies]
+python = "^3.9"
+attrs = "18.2.0"
+click = "7"
+toolz = "0.9.0"
+tqdm = "4.32.1"
+pysunspec = "2.1.1"
+setuptools = "53.0.0"
+toml = "^0.10.2"
+[tool.poetry.dev-dependencies]
+gitignoreio = "0.1.5"
+pyinstaller = "4.2"
+
+
+[tool.poetry.scripts]
+epcsunspecdemo = 'epcsunspecdemo.cli:cli'
+
+[build-system]
+requires = ["setuptools", "poetry-core>=1.0.0"]
```

In the poetry dynamic versioning docs, I see the following line:

```suggestion
requires = ["setuptools", "poetry-core>=1.0.0", "poetry-dynamic-versioning"]
```

Not sure if this is necessary? Does the dynamic versioning work?